fitbenchmarking/fitting_benchmarking.py

```diff
@@ -88,19 +88,20 @@ def do_fitting_benchmark(nist_group_dir=None, cutest_group_dir=None, neutron_dat
             os.makedirs(group_results_dir)
 
 
-        prob_results = [do_fitting_benchmark_group(group_name, group_results_dir, problem_block,
-                                                   minimizers, use_errors=use_errors)
-                        for problem_block in problem_groups[group_name]]
+        prob_results = (do_fitting_benchmark_group(group_name, group_results_dir,
+                                                  problem_block, minimizers,
+                                                  use_errors=use_errors)
+                       for problem_block in problem_groups[group_name])
 
     probs, results = list(zip(*prob_results))
```

Comment:
please add comment explaining this line

Reply:
Also, could you add @returns the the method description of this file please

Reply:
Testing another PR at the moment, will deal with this an the other PR merge issues in a bit.

Reply:
If you are referring to `probs, results = list(zip(*prob_results))` , this will be removed in #74 so I don't see the point of explaining it.